mobile/mobile.go

```diff
@@ -59,17 +59,6 @@ func (w *Wrapper) Start() error {
 	return nil
 }
 
-func (w *Wrapper) StartGateway() error {
-	if w.gatewayRunning {
-		return nil
-	}
-	if _, err := tcore.ServeHTTPGatewayProxy(w.node); err != nil {
-		return err
-	}
-	w.gatewayRunning = true
-	return nil
-}
-
 func (w *Wrapper) Stop() error {
 	return w.node.Stop()
```

Comment:
we'll need to catch the `ErrNodeNotRunning` error here, right? i can do in my branch though

Reply:
(since we don't want to error out in RN)